bokeh/server/tests/test_utils.py

```diff
@@ -26,25 +24,25 @@ def helper():
     return wait_until(helper)
 
 
-def wait_redis_gone(port):
-    def helper():
-        client = redis.Redis(port=port)
-        try:
-            client.ping()
-            return False
-        except redis.ConnectionError:
-            return True
-    return wait_until(helper)
+# def wait_redis_gone(port):
+#     def helper():
+#         client = redis.Redis(port=port)
+#         try:
+#             client.ping()
+#             return False
+#         except redis.ConnectionError:
+#             return True
+#     return wait_until(helper)
```

Comment:
@almarklein do you want to remove these, or else I can? I think as soon as they are removed then we can merge this.